README.md

```diff
@@ -1,99 +1,110 @@
-Optimizely Java SDK
-===================
+# Optimizely Java SDK
+
 [![Build Status](https://travis-ci.org/optimizely/java-sdk.svg?branch=master)](https://travis-ci.org/optimizely/java-sdk)
+
```

Comment:
Please add coveralls badge if easy to do so, or create a follow-up story

Reply:
Also, this new line is causing the two badges you current have to appear on two lines. Please remove so that they're all on the same line.

Reply:
> Please add coveralls badge if easy to do so, or create a follow-up story

Coverall links disconnected for some SDKs including java. Let's review them all together. @russell-loube-optimizely